roles/ceph-osd/tasks/scenarios/non-collocated.yml

```diff
@@ -70,6 +80,11 @@
     - not containerized_deployment
     - item.0.partitions|length == 0
 
+  register: osd
```

Comment:
This won't work because of the `changed_when: false`.

Reply:
why is this task set to changed_when false?   Can we change it?  It isn't present in the same command for manually prepare ceph.

Reply:
nvm I'll work around it